core/internal/cltest/cltest.go

```diff
@@ -539,6 +543,12 @@ func (ta *TestApplication) NewBox() packr.Box {
 	return packr.NewBox("../fixtures/operator_ui/dist")
 }
 
+func (ta *TestApplication) MustStartAndStop() {
+	ta.t.Helper()
+	require.NoError(ta.t, ta.Start())
+	ta.t.Cleanup(func() { require.NoError(ta.t, ta.Stop()) })
```

Comment:
Notice the old cleanup func at 460 was calling `StopIfStarted()` which is not overridden by `TestApplication`, so only the `ChainlinkApplication` was stopped. Solely using this for cleanup leaks `TestApplication` resources `Store`,`Server`, and `wsServer` which are cleaned up in the overridden `Stop()`.